examples/host/simpleResolver/README.md

```diff
@@ -91,3 +89,9 @@ ASCII art for the win!
 ```bash
 python ./simpleResolver.py locatableContent bal:///cat | jq -r '.locatableContent.url' | xargs jp2a
 ```
+
+[^1]: Manually installing BAL:
+   Clone the repo:
+  `git clone https://github.com/OpenAssetIO/OpenAssetIO-Manager-BAL.git`
+  Extend the plugin search paths:
+  `export OPENASSETIO_PLUGIN_PATH=./OpenAssetIO-Manager-BAL/plugin`
```

Comment:
[Consider] Perhaps "Set the plugin search paths:", as the code below doesn't extend it as such. Either that or have the code read `export OPENASSETIO_PLUGIN_PATH=.$OPENASSETIO_PLUGIN_PATH;/OpenAssetIO-Manager-BAL/plugin`

Reply:
Hows [this](https://github.com/OpenAssetIO/OpenAssetIO/pull/839/commits/005133e656ea4ec7ca393c38786f8e2b3773e251).